code/SecureFileController.php

```diff
@@ -139,12 +145,16 @@ public function sendFile($file) {
 			return file_get_contents($path);
 		}
 
-		$disposition = $this->config()->content_disposition;
-		if(!$disposition) $disposition = 'attachment';
+		if(!$disposition) {
+			$disposition = $this->config()->content_disposition;
+			if (!$disposition) $disposition = 'attachment';
+		}
+
+		if(!$name) $name = basename($path);
```

Comment:
Curly braces needed

Reply:
Well, I stayed within the coding standard of the file, the whole module could use an update. But I'll update it.